pkg/workflow/controller/handlers/workflowtrigger/cron_job.go

```diff
@@ -16,6 +18,11 @@ import (
 
 const (
 	KeyTemplate = "%s/%s"
+
+	// the time zone difference, in minutes, from current locale (host system settings) to UTC
```

Comment:
`current locale (host system settings)` it's misleading.  And `time zone difference` seems not a precise description.

Reply:
I copy from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTimezoneOffset